Common/syntheaPostInstall.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+mkdir /tmp/loader
+pushd /tmp/loader
+curl -fsSL --progress-bar https://github.com/OSEHRA/VistA-FHIR-Data-Loader/releases/download/0.1/VISTA_SYNTHETIC_DATA_LOADER_BUNDLE_0P2T1.KID.zip -o loader.zip
```

Comment:
is there a way to use the github api to always get the latest version? may prevent future updates to this script.

Reply:
No. We have to use specific KIDS builds in releases. We are not just getting the source code as there is some pre-install and post-install stuff that needs to be done.

Reply:
I'm referring to the latest release kids build not source code. Not a big issue, if we lock versions then we should probably have a comment here.